12-hooks-in-react/src/components/Search.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+
+const Search = () => {
+  const [term, setTerm] = useState("");
+
+  useEffect(()=>{
+    console.log('asdlfkj');
+  });
+
+  return (
+    <div>
```

Comment:
Is this outside div necessary?

Reply:
Will remove the unnecessary <div> when the Search component is finally in working condition.